tests/Flow/DebounceTaskCancellation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OwlCoreFlow = OwlCore.Flow;
+
+namespace OwlCore.Tests.Flow
+{
+    [TestClass]
+    public class DebounceTaskCancellation
+    {
+        [TestMethod, Timeout(5000)]
```

Comment:
(no penalty) Since this type of test restricts internal behavior for performance reasons (rather than the usual public member test), let's give this test a `[TestCategory]` attribute.

```suggestion
        [TestMethod, Timeout(5000), TestCategory("PerformanceCheck")]
```

Reply:
@amaid This was added to the old `DebounceTaskCancellation.cs` file, and is missing from the new file.

Reply:
This is fixed.